### DISCUSSION TITLE
Krotov propagator vs qutip mesolve

### DISCUSSION THREAD

**Discussion by Leeseok-628** [qucontrol/krotov#107] [Q&A] [answered]
If I use optimized pulse sequences done via krotov optimization to evolve the same Hamiltonian through qutip.mesolve, the final state seems very different from what I got earlier from the Krotov optimization. They're pretty much the same in 1~3 number of qubit systems, but begin to be different when it's more than 4+ qubits. I assume that this is because evolution methods of Krotov propagator is different from the mesolve function. How can I alleviate the issue? Does increasing number of timesteps help?

**Comment by goerz** [comment, marked as answer]
> Does increasing number of timesteps help?

Yes
